unifi/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN \
         openjdk-8-jdk-headless=8u342-b07-0ubuntu1~20.04 \
     \
     && curl -J -L -o /tmp/unifi.deb \
-        "https://dl.ui.com/unifi/7.1.68/unifi_sysvinit_all.deb" \
+        "https://dl.ui.com/unifi/7.2.91-85c367a4aa/unifi_sysvinit_all.deb" \
```

Comment:
This is a release candidate, and not a stable release.
We don't ship RCs in the add-on.